src/app/components/Statistics/BlockchainDataCharts/index.tsx

```diff
@@ -67,6 +67,7 @@ const BlockchainCharts = ({
     const [openModal, setOpenModal] = useState(false)
     const [startDate, setStartDate] = useState<Date>()
     const [endDate, setEndDate] = useState<Date>(new Date())
+    const [seeTimeAxis, setSeeTimeAxis] = useState<String>('month')
```

Comment:
can we store the time selected in the store and with that we can memorize which time was selected